ci/test.sh

```diff
@@ -148,7 +148,7 @@ EOF
     if (( ${CPP:-0} )); then
         td="$(mktemp -d)"
 
-        git clone --depth 1 https://github.com/japaric/hellopp "${td}"
+        git clone --depth 1 https://github.com/dalance/hellopp -b update_cc "${td}"
```

Comment:
Should we fork this in the `cross-rs` org instead?

Reply:
I think so.